src/Command/BaseSynchronizeCommand.php

```diff
@@ -34,11 +34,13 @@ public function __construct(EntityManagerInterface $entityManager)
         $this->entityManager = $entityManager;
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         // disable logger because of memory issues
         $this->entityManager->getConfiguration()->setSQLLogger(null);
         $this->entityManager->getConnection()->getConfiguration()->setSQLLogger(null);
         gc_enable();
+
+        return 1;
```

Comment:
this indicates that an error happened, right?

Reply:
🙈 your right